apps/demo/src/styles.scss

```
@@ -402,39 +402,48 @@
-    --ngx-signal-form-fieldset-muted-color: var(--color-text-muted);
-  }
-
   .choice-group-fieldset__legend {
-    @apply block text-sm font-normal tracking-wide;
-    padding-inline-start: 0.125rem;
-    color: var(--color-text-muted);
+    @apply block;
+    font-size: var(
+      --ngx-signal-form-fieldset-selection-legend-font-size,
+      0.875rem
+    );
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)